pkg/pgmodel/migrations/sql/idempotent/base.sql

```diff
@@ -1003,11 +1013,22 @@ BEGIN
                 timescaledb.compress_segmentby = 'series_id',
                 timescaledb.compress_orderby = 'time, value'
             ); $$, metric_table_name);
+
         --chunks where the end time is before now()-1 hour will be compressed
-        PERFORM add_compress_chunks_policy(format('SCHEMA_DATA.%I', metric_table_name), INTERVAL '1 hour');
+        IF new_compress_policy THEN
+            PERFORM add_compression_policy(format('SCHEMA_DATA.%I', metric_table_name), INTERVAL '1 hour');
+        ELSE
+            PERFORM add_compress_chunks_policy(format('SCHEMA_DATA.%I', metric_table_name), INTERVAL '1 hour');
+        END IF;
     ELSE
-        PERFORM remove_compress_chunks_policy(format('SCHEMA_DATA.%I', metric_table_name));
+        IF new_compress_policy THEN
```

Comment:
```suggestion
       IF SCHEMA_CATALOG.get_timescale_major_version() >= 2 THEN
```

Reply:
I thought it was too much last week when I tried it, but I like it better now 🤔